php/helpers.php

```diff
@@ -68,17 +68,38 @@ function block_field( $name, $echo = true ) {
 			$value = __( 'No', 'block-lab' );
 		}
 
+		/*
+		 * The user value is stored as the slug (user_login), which cannot change.
+		 * But it's echoed as the display_name, which can change via a setting in /wp-admin/user-edit.php.
+		 */
+		if ( has_user_control_type( $block_lab_config['fields'][ $name ] ) ) {
```

Comment:
Instead of creating a new function `has_user_control_type()`, we could do this:

Just below where `$value` is defined on line 33—36, we could define a new variable called `$field`:
```
if ( array_key_exists( $name, $block_lab_config['fields'] ) ) {
    $field = new Field( $block_lab_config['fields'][ $name ] );
} else {
    $field = new Field();
}
```

Once we have that, we don't need that additional helper function anymore, because to check if it's a `user` control type, we can just do:

```
if ( 'user' === $field->control ) {
    …
}
```

I don't know for sure that `$block_lab_config['fields'][ $name ]` is properly mapped to the `$args` a `Field` constructor is expecting, but if it is that would be neat.

Reply:
Good idea, https://github.com/getblocklab/block-lab/pull/233/commits/c9c0476380c5433b420c5fb831135cb39b7e3955 uses your code above.